miso-web/src/main/webapp/scripts/hot_sample.js

```diff
@@ -36,6 +36,10 @@ HotTarget.sample = (function() {
       return item.label == flatObj.identityAlias;
     }, flatObj.potentialIdentities);
   };
+  
+  var getExternalNames = function(identityAlias){
+    return new Set(identityAlias.replace(/.*--(\s*)/, "").toLowerCase().split(","));
```

Comment:
I have some concerns about using `Set` here, as it isn't well-supported in older browsers like IE11 and MISO doesn't have a transpilation step. Lab computers are often older (our lab's computer has IE11 installed). Could you refactor this to be less elegant? :stuck_out_tongue_closed_eyes: 
Alternatively, you could add this check to identities when they're being created or updated, so that they can never come back to the client in this duplicated state.